client/src/components/RequestManagement/RequestManagementBoard.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {Component} from 'react';
+import {Card, Container, Grid} from "semantic-ui-react";
+import RequestedHardwareItem from "./RequestedHardwareItem";
+import {RequestedItem} from "../inventory/HardwareItem";
+import {RequestedItemsByUser} from "./RequestedItemsByUser";
+import {UserItemList} from "../RequestManagementContainer";
+
+
+interface RequestManagementBoardProps {
+    title: string,
+    boardStyle: any,
+    items: RequestedItem[],
+    sortedItems: UserItemList[],
+    sortByUsers: boolean,
+    backToPrep: boolean,
+    handleApprove: (id: number) => void,
+    handleDecline: (id: number) => void,
+    handleReady: (user: string) => void,
+    handleCross: (user: string) => void,
+    handlePrep: (user: string) => void,
+    handleDone: (user: string) => void,
+}
+
+class RequestManagementBoard extends Component<RequestManagementBoardProps, {}> {
+    render() {
+        const returnValue = this.props.sortByUsers ? (
+            <Grid.Column>
+                <h2>{this.props.title}</h2>
+                <Container placeholder style={this.props.boardStyle}>
+                    <Card.Group>
+                        {this.props.sortedItems.map((item, index) => {
+                            return (
+                                <RequestedItemsByUser
+                                    key={index}
```

Comment:
[Don't use a list index as a key since it's not guaranteed to be unique.](https://reactjs.org/docs/lists-and-keys.html#keys)  Should be the request ID provided by the server (but you can just mock the value like everything else for now)